rkt/app_add.go

```diff
@@ -53,42 +53,47 @@ func init() {
 func runAppAdd(cmd *cobra.Command, args []string) (exit int) {
 	if len(args) < 2 {
 		stderr.Print("must provide the pod UUID and an IMAGEID")
-		return 1
+		return 254
 	}
 
 	err := parseApps(&rktApps, args[1:], cmd.Flags(), true)
 	if err != nil {
 		stderr.PrintE("error parsing app image arguments", err)
-		return 1
+		return 254
```

Comment:
We should yes, but a bit out of scope here (we basically need to touch all stage0 source files).